health-checker/Gruntfile.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = function (grunt) {
```

Comment:
grunt? haven't used this in years. i thought web pack pretty much killed off grunt?

Reply:
Check out https://github.com/hubotio (e.g.: https://github.com/hubotio/hubot-help). Since the build tool only needs to be able to run the tests and publish, it seems npm can do the job with a couple of dev dependencies.

Reply:
Also, based on this at this moment I think separate repos might be better: https://github.com/hubotio

Reply:
Although, the official documentation suggests the yeoman generator: https://hubot.github.com/docs/scripting/#creating-a-script-package

Reply:
I see a lot of the official script packages don't use grunt. Rather they use `chai` and `npm test`
example https://github.com/hubotio/hubot-diagnostics/blob/master/package.json#L29

In my package, im going to give this a shot

Reply:
Yepp, that's what I tried to say by:
> Since the build tool only needs to be able to run the tests and publish, it seems npm can do the job with a couple of dev dependencies.

Chai is just for assertions nyc (from istanbul.js) seems to be the tool who runs the tests instead of grunt. I'm pretty curious, I think I'll wait merging my PR to see if it is easy to replace Grunt. :)

```
"test": "nyc --reporter=html --reporter=text mocha",
"coverage": "nyc report --reporter=text-lcov | coveralls",
"semantic-release": "semantic-release pre && npm publish && semantic-release post"
```

Reply:
If we're good with this initial implementation I would like to merge it. Any objections?

Reply:
Go ahead :)